src/interfaces/jira.ts

```diff
@@ -158,6 +159,16 @@ export interface JiraAssociation {
 	values: string[];
 }
 
+export interface JiraCommitKey {
+	commitHash: string;
+	repositoryId: string;
+}
+
+export interface JiraCommitAssociation {
```

Comment:
Instead of creating 2 types for the same association, why not just include both in the same?
```
interface JiraAssociation {
   associationType: string;
   values: string[] | JiraCommitKey[];
}
```